### PR TITLE
feat(ProgressBarWindows) - add ProgressBarWindows!

### PR DESCRIPTION
**Pull request by quincycs**
https://twitter.com/quincymitch/status/722409885187575808

**Motivation**
https://github.com/ReactWindows/react-native/issues/185

I looked at ProgressBarAndroid's source and provided the same behavior as android.
**Test plan**

I tested the below working in addition to each of these values changing after a setState / re-render.
<ProgressBarWindows indeterminate={false} progress={.5} color="black"/>

### CODE REVIEW

**Comment by mpodwysocki** [comment]
@quincycs first, thank you very much for the Pull Request.  In order to proceed, can you sign the [Microsoft CLA](https://cla.microsoft.com/) which allows us to accept code from the outside.

**Comment by quincycs** [comment]
Ok will look into signing.  @modwysocki  I just noticed I forked and pull requested on the WindowsStore branch.  Is that ok?  I'm guessing the ReactWindows branch is the preferred branch to fork and merge.

**Comment by quincycs** [comment]
@mpodwysocki oops misspelled name :)

File: Libraries/react-native/react-native.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ var ReactNative = {
   get DatePickerIOS() { return require('DatePickerIOS'); },
   get DrawerLayoutAndroid() { return require('DrawerLayoutAndroid'); },
   get FlipViewWindows() { return require('FlipViewWindows'); },
+  get ProgressBarWindows() {return require('ProgressBarWindows');},
   get Image() { return require('Image'); },
```
Comment (rozele) [inline Libraries/react-native/react-native.js]: Co-locate this ProgressBarAndroid and ProgressViewIOS.

**Comment by rozele** [comment]
+1, thanks @quincycs!

**Comment by rozele** [comment]
@quincycs, we just rebased our ReactWindows branch off the master branch @ [http://github.com/facebook/react-native](http://github.com/facebook/react-native) yesterday.  Could you re-apply your changes to the ReactWindows branch?

**Comment by mpodwysocki** [comment]
@quincycs yes, ReactWindows is now the preferred branch to merge.  The WindowsStore branch was just there to demonstrate if you wanted to build the F8 App.

**Comment by quincycs** [comment]
ok thx!  Working with the powers that be to get the CLA agreement signed, then will submit another PR.  Curious...why did the AppVeyor build fail?  (Did i do everything right except for the branch)

**Comment by mattpodwysocki** [comment]
@quincycs don't worry about the build failure, that's our issue with AppVeyor which we hope to fix shortly.  Shall we close this one then?

**Comment by rozele** [comment]
@quincycs: I hope you don't mind, but I used your code to make a new PR.  I wanted to get the XHRExample with the ProgressBar working in UIExplorer.  I made sure to set you as the --author of the commit.

**Pull request closed without merging**